apps/condo/domains/organization/schema/RegisterNewOrganizationService.test.js

```diff
@@ -8,6 +8,13 @@ const { registerNewOrganization } = require('@condo/domains/organization/utils/t
 const { OrganizationEmployee, OrganizationEmployeeRole } = require('../utils/testSchema')
 const { ServiceSubscription } = require('@condo/domains/subscription/utils/testSchema')
 
+const { DEFAULT_ROLES } = require('@condo/domains/organization/constants/common.js')
+
+const getPermissions = (roleName) => {
+    const excludeFields = ['name', 'description']
```

Comment:
looks like static constant, that can be moved outside the function

Reply:
Done